src/main/java/br/com/sbk/sbking/core/boardrules/bridgeopenings/DealerHasOneNoTrumpOpeningBoardRule.java

```diff
@@ -0,0 +1,25 @@
+package br.com.sbk.sbking.core.boardrules.bridgeopenings;
+
+import br.com.sbk.sbking.core.Board;
+import br.com.sbk.sbking.core.BoardRule;
+import br.com.sbk.sbking.core.Hand;
+import br.com.sbk.sbking.core.HandEvaluations;
+
+public class DealerHasOneNoTrumpOpeningBoardRule implements BoardRule {
+
+  @Override
+  public boolean isValid(Board board) {
+    Hand dealerHand = board.getHandOf(board.getDealer());
+    HandEvaluations handEvaluations = dealerHand.getHandEvaluations();
+    return hasCorrectHCPRange(handEvaluations) && hasCorrectDistribution(handEvaluations);
```

Comment:
Also it can't have rich suit 5º. It doesn't seems that you included this condition.